docs/gestures.md

```diff
@@ -222,6 +227,26 @@ In these examples, note that the element parameter is always optional.
   js.executeScript("mobile: swipe", swipeObject);
   ```
   
+### Scroll
+
+* **WD.js:**
+
+  ```js
+  // scroll the view down
+  driver.execute("mobile: scroll", [{direction: 'down'}], function(err) {
+    // continue testing
+  });
+  ```
+
+* **Java:**
+
+  ```java
+  JavascriptExecutor js = (JavascriptExecutor) driver;
+  HashMap<String, Double> scrollObject = new HashMap<String, Double>();
+  scrollObject.put("direction", "down");
```

Comment:
Your HashMap is declared with < String, Double> and here your are passing < String, String> .